Source/Scene/Expression.js

```diff
@@ -0,0 +1,1886 @@
+
+    function isVariable(name) {
+        return (name.substr(0, 4) === 'czm_');
+    }
+
+    function getPropertyName(variable) {
+        return variable.substr(4);
+    }
+
+    function createRuntimeAst(expression, ast) {
+        var node;
+        var op;
+        var left;
+        var right;
+
+        if (ast.type === 'Literal') {
+            node = parseLiteral(ast);
+        } else if (ast.type === 'CallExpression') {
+            node = parseCall(expression, ast);
+        } else if (ast.type === 'Identifier') {
+            node = parseKeywordsAndVariables(ast);
+        } else if (ast.type === 'UnaryExpression') {
+            op = ast.operator;
+            var child = createRuntimeAst(expression, ast.argument);
+            if (unaryOperators.indexOf(op) > -1) {
+                node = new Node(ExpressionNodeType.UNARY, op, child);
+            } else {
+                //>>includeStart('debug', pragmas.debug);
+                throw new DeveloperError('Unexpected operator "' + op + '".');
+                //>>includeEnd('debug');
+            }
+        } else if (ast.type === 'BinaryExpression') {
+            op = ast.operator;
+            left = createRuntimeAst(expression, ast.left);
+            right = createRuntimeAst(expression, ast.right);
+            if (binaryOperators.indexOf(op) > -1) {
+                node = new Node(ExpressionNodeType.BINARY, op, left, right);
+            } else {
+                //>>includeStart('debug', pragmas.debug);
+                throw new DeveloperError('Unexpected operator "' + op + '".');
+                //>>includeEnd('debug');
+            }
+        } else if (ast.type === 'LogicalExpression') {
+            op = ast.operator;
+            left = createRuntimeAst(expression, ast.left);
+            right = createRuntimeAst(expression, ast.right);
+            if (binaryOperators.indexOf(op) > -1) {
+                node = new Node(ExpressionNodeType.BINARY, op, left, right);
+            }
+        } else if (ast.type === 'ConditionalExpression') {
+            var test = createRuntimeAst(expression, ast.test);
+            left = createRuntimeAst(expression, ast.consequent);
+            right = createRuntimeAst(expression, ast.alternate);
+            node = new Node(ExpressionNodeType.CONDITIONAL, '?', left, right, test);
+        } else if (ast.type === 'MemberExpression') {
+            node = parseMemberExpression(expression, ast);
+        } else if (ast.type === 'ArrayExpression') {
+            var val = [];
+            for (var i = 0; i < ast.elements.length; i++) {
+                val[i] = createRuntimeAst(expression, ast.elements[i]);
+            }
+            node = new Node(ExpressionNodeType.ARRAY, val);
+        }
+        //>>includeStart('debug', pragmas.debug);
+        else if (ast.type === 'Compound') {
+            // empty expression or multiple expressions
+            throw new DeveloperError('Provide exactly one expression.');
+        }  else {
```

Comment:
`else` is not needed; just `throw` and use `// jshint ignore:line` below.

Also note that this branch will need some tweaks if #5344 lands in master first.